_includes/join-section.html

```diff
@@ -1,6 +1,6 @@
 <section class="join-section">
   <img src="/assets/img/favicons/mstile_home.png" class="join-img">
-  <h2 class="title">Join our Riot, Telegram and Discord Channels</h2>
+  <h2 class="title">Join our community and be part of the revolution</h2>
```

Comment:
I'd omit the "revolution". Feels like every blockchain project is revolutionary (based on their promo) and it's laughable in majority of cases.. I'd keep our feet on the ground :)

Reply:
Aha yeah, I hesitated about this one and I ultimately agree.

Reply:
I'd like to keep it short and to the point - maybe remove the channel names, as it's obvious (and listed below in buttons?).. I almost want to have there just:
> Join our community
> Join our passionate community

or
> Join our community of privacy pioneers

Reply:
> Join our community

Alright then let's keep it like that. I like "of privacy pioneers" as well, but the community is broader than that (ecommerce fans, crypto fans, passive income fans, etc)